ContactList/ContactList/Model.swift

```diff
@@ -13,6 +13,8 @@ struct Constants {
     static let emptyAvatar = UIImage(named: "emptyAvatar")!
     static let validColor = UIColor.white
     static let invalidColor = UIColor.red
+    static let grayTextColor = [NSAttributedString.Key.foregroundColor: UIColor.gray]
```

Comment:
As it's NSAttributedString attribute, please rename these two fields into `grayColorAttribute` and `blackColorAttribute`
Otherwise property type is not clear.